catroid/src/org/catrobat/catroid/content/bricks/WhenNfcBrick.java

```diff
@@ -80,6 +80,10 @@ public WhenNfcBrick(WhenNfcScript script) {
 		this.oldSelectedNfcTag = null;
 		this.nfcTag = script.getNfcTag();
 		this.whenNfcScript = script;
+
+		if (script != null && script.isCommentedOut()) {
+			setCommentedOut(true);
+		}
```

Comment:
Ahhh, now I see your Problem ;) I think you are right that this cannot be done without editing every Brick class. This is really a tough one ;D Maybe you could have a method in BrickBaseType called `setMembers(BrickBaseType copyBrick)` where you set all Members the same as in copyBrick. Then in every "SpecialBrick" you call super.clone() and after that clonedBrick.setMembers(this). You still need to change every Brick class, but whenever someone adds a new Member in BrickBaseType, he/she just needs to add one line in the setMember method.
I am not perfectly sure, but I think this should be the easiest way with the smallest amount of work, especially for the future while preserving a quite easy to read and clean code. But it would be very nice if someone has a better idea ;)